### PR TITLE
Set tags on log groups

### LINKED ISSUE #1078: Tags are not propagated to API log groups
### Description

Tags specified in cluster configuration are not propagated to API log groups because API log groups are created by FluentD.

#### Suggested solution

- explore ways to add tags to log groups created by FluentD
- create log groups upon API creation in operator

### PR DESCRIPTION
**Pull request by tthebst**
closes #1078

---

How do you format yaml's? My VSCode formats them like this.

checklist:

- [x] run `make test` and `make lint`
- [x] test manually (i.e. build/push all images, restart operator, and re-deploy APIs)
- [ ] update examples
- [ ] update docs and add any new files to `summary.md` (view in [gitbook](https://docs.cortex.dev/v/master) after merging)
- [ ] cherry-pick into release branches if applicable
- [ ] alert the dev team if the dev environment changed

### CODE REVIEW

**Comment by deliahu** [comment]
We haven't been using a YAML formatter, but maybe we should! Which one do you use?

Regardless, I think we should make a separate PR after this one if we do decide to reformat, to keep the diff in the PR more relevant

File: manager/install.sh [hunk 1]
```diff
@@ -238,6 +238,7 @@ function main() {
   echo "✓"
 
   echo -n "￮ configuring logging "
+  export CORTEX_TAGS_JSON=$(echo $CORTEX_TAGS  | jq -sR '[split(",")[] | rtrimstr("\n") | split("=") | {(.[0]): .[1]} ] | add' | tr -d '\n' )
```
Comment (deliahu) [inline manager/install.sh:241]: Nice job with the `jq` query! I have always struggled to write those :)

There is a simpler place we can do this, which is in `cluster_config_env.py` (that is the file which exports these env vars. Right below `print(f"export CORTEX_TAGS={inlined_tags}")`, we should be able to add a line like `print(f"export CORTEX_TAGS_JSON='{json.dumps(value)}'")`, what do you think about this approach?

File: manager/manifests/fluentd.yaml [hunk 2]
```diff
@@ -168,9 +169,6 @@ metadata:
   name: fluentd
   namespace: default
 spec:
-  selector:
```
Comment (deliahu) [inline manager/manifests/fluentd.yaml:171]: Good catch! Do you mind moving this to a separate PR, along with the change in `operator.yaml`?

File: manager/install.sh (hunk 1 above)
Comment (tthebst) [reply manager/install.sh:241]: everything is better than jq lol
